lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -572,6 +565,13 @@ void HdVP2BasisCurves::Sync(
 
     const SdfPath& id = GetId();
 
+    // Update the selection status if it changed.
```

Comment:
This code was in initRepr, but it is not really any different from other code that has dirty flags so there is not a reason for it to be there.